lib/rules/spaced-comment.js

```diff
@@ -281,26 +328,36 @@ module.exports = {
 
             // Checks.
             if (requireSpace) {
-                if (!rule.regex.test(node.value)) {
+                if (!rule.beginRegex.test(node.value)) {
```

Comment:
i think `beginMatch` and `endMatch` might work better, since "matched" implies a boolean also